editor/plugins/script_editor_plugin.h

```diff
@@ -488,6 +487,7 @@ class ScriptEditor : public PanelContainer {
 
 	void get_breakpoints(List<String> *p_breakpoints);
 
+	ScriptEditorBase *get_current_editor() const;
```

Comment:
Please don't expose this, you can call `edit` with the resource to go to a specific line.

Reply:
No problem, I'll drop the commit 👍